AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/InviteController.cs

```diff
@@ -0,0 +1,154 @@
+            });
+        }
+
+        // POST: Admin/Invite/SendCampaignManagerInvite/5
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [Route("[area]/[controller]/[action]/{campaignId:int}")]
+        public async Task<IActionResult> SendCampaignManagerInvite(int campaignId, InviteViewModel invite)
+        {
+            if (invite == null)
+            {
+                return BadRequest();
+            }
+
+            if (ModelState.IsValid)
+            {
+                var campaign = await _mediator.SendAsync(new CampaignByCampaignIdQuery { CampaignId = invite.CampaignId });
+
+                if (campaign == null)
+                {
+                    return BadRequest();
+                }
+
+                if (!IsUserAuthorizedToSendInvite(campaign.ManagingOrganizationId))
+                {
+                    return Unauthorized();
+                }
+
+                var user = await _userManager.GetUserAsync(User);
+                await _mediator.SendAsync(new CreateInviteCommand { Invite = invite, UserId = user?.Id });
+
+                return RedirectToAction(actionName: "Details", controllerName: "Campaign", routeValues: new { area = "Admin", id = invite.CampaignId });
+            }
+
+            return View("Send", invite);
+        }
+
+        // POST: Admin/Invite/SendEventManagerInvite/5
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [Route("[area]/[controller]/[action]/{eventId:int}")]
+        public async Task<IActionResult> SendEventManagerInvite(int eventId, InviteViewModel invite)
+        {
+            if (invite == null)
+            {
+                return BadRequest();
+            }
+
+            if (ModelState.IsValid)
+            {
+                var @event = await _mediator.SendAsync(new EventByEventIdQuery { EventId = invite.EventId });
+                if (@event == null)
+                {
+                    return BadRequest();
+                }
+
+                if (!IsUserAuthorizedToSendInvite(@event.Campaign.ManagingOrganizationId))
+                {
+                    return Unauthorized();
+                }
+
+                var user = await _userManager.GetUserAsync(User);
+                await _mediator.SendAsync(new CreateInviteCommand { Invite = invite, UserId = user?.Id });
+
+                return RedirectToAction(actionName: "Details", controllerName: "Event", routeValues: new { area = "Admin", id = invite.EventId });
+            }
+
+            return View("Send", invite);
+        }
+
+        private bool IsUserAuthorizedToSendInvite(int organizationId)
+        {
+            if (!User.IsOrganizationAdmin())
+            {
+                return false;
+            }
+
+            var userOrganizationId = User.GetOrganizationId();
+
+            if (!userOrganizationId.HasValue || organizationId != userOrganizationId)
+            {
+                return false;
+            }
+            return true;
```

Comment:
it looks like in general, you're trying to see if the user is an org admin of the organiztaionId passed into this method. There is an extension method for that:
`ClaimsExtensions.IsOrganizationAdmin(this ClaimsPrincipal user, int organizationId)`

your code can be:
```
if (!User.IsOrganizationAdmin(organiaztionId))
{
    return false;
}
```

for this method, and that's it.